docs/installation.rst

```diff
@@ -29,4 +29,13 @@ To install ``jdaviz`` for development or from source in editable mode::
    cd jdaviz
    pip install -e .
 
+To enable the hot reloading of vue templates, install `watchdog`::
+
+   pip install watchdog
+
+And to the top of a notebook add::
+
+   import ipyvue
+   ipyvue.watch('../jdaviz')
```

Comment:
The following works in general (including when using an installed `jdaviz` without having to worry about getting the path right manually):

```
import jdaviz
import os
import ipyvue
ipyvue.watch(os.path.dirname(jdaviz.__file__))
```

Should we consider a `jdaviz.enable_hot_reloading()` which wraps the above and could also handle checking for the `watchdog` optional dependency?

Reply:
yeah, good idea 👍. Will add it.

I will also look into the Voila problem, I think I know what's causing it.